### DISCUSSION TITLE
E3 Vitocal Circulation pump

### DISCUSSION THREAD

**Discussion by cb0b** [MyHomeMyData/ioBroker.e3oncan#80] [Q&A]
Hallo,
weiss jemand wo in dem CAN Bus Objekt Baum der Zustand der Heizungszirkulationspumpe versteckt ist. Die Viessmann API Version bietet den Zustand an und der muss ja irgendwo auslesbar sein - oder wird der aus einem anderen Punkt erzeugt?
Im Homeassistant 2024.10.4 steht der Zustand in binary_sensor.e3_vitocal_circulation_pump. Die Anlage ist eine Virocal 222-S via externem Canbus ausgelesen.

**Comment by MyHomeMyData** [comment]
Das weiß ich auch nicht. Interessante Kandidaten sind vielleicht:
401_MixerOneCircuitPump
1241_MixerOneCircuitPumpMode
2792_MixerOneCircuitPumpStatus
2896_PrimaryPumpOneStatus
Evtl. hast Du Erfolg, wenn Du die API-Werte mit diesen Datenpunkten abgleichst.
